stdlib/2/__builtin__.pyi

```diff
@@ -718,8 +718,8 @@ def intern(string: str) -> str: ...
 def iter(iterable: Iterable[_T]) -> Iterator[_T]: ...
 @overload
 def iter(function: Callable[[], _T], sentinel: _T) -> Iterator[_T]: ...
-def isinstance(o: object, t: Union[type, Tuple[type, ...]]) -> bool: ...
-def issubclass(cls: type, classinfo: Union[type, Tuple[type, ...]]) -> bool: ...
+def isinstance(o: object, t: Union[type, Tuple[Union[type, Tuple], ...]]) -> bool: ...
```

Comment:
@pkch @gvanrossum 
I think here and below it should be ``Tuple[type, ...]`` no just ``Tuple`` at the end.
Otherwise ``isinstance(obj, ((1, 2), (3, 4)))``will be valid.

This also applies to Python 3 stub.